### DISCUSSION TITLE
redirect http to https

### DISCUSSION THREAD

**Discussion by gsgalezowski** [ArchiveBox/ArchiveBox#868] [General]
How to make Archivebox accessible via https? Is it possible to do a redirect from http to https?

**Comment by pirate** [comment]
You need to put a reverse proxy in front of it like nginx, caddy, cloudflare, etc. to do the SSL termination. An example nginx config is provided already: https://github.com/ArchiveBox/ArchiveBox/blob/dev/etc/nginx.conf + https://github.com/ArchiveBox/ArchiveBox/blob/dev/docker-compose.yml#L57

Or for something simpler, use Caddy or Cloudflare in front of it: https://caddyserver.com/docs/quick-starts/reverse-proxy
